README.md

```diff
@@ -38,21 +56,53 @@ For expanded descriptions of the datasets, please see the Pediatric Cancer Data
+#### Somatic CNV Calling
+
+Copy number variants are called using multiple tools: ControlFreeC, CNVkit, and GATK CNV. 
+CNVkit calls are adjusted for sex using a pre-calculated germline sex estimate. 
+Manta is used to call structural variants (SVs). 
+All calls are made using GRCh38 references and variants are then annotated with gnomAD frequencies.
+Publicly available files are further subjected to “germline masking,” which removes low frequency variants that could be used to identify the sample donor. 
+For more details see the GitHub release at [Kid's First Somatic Variant Workflow](https://github.com/kids-first/kf-somatic-workflow) or to run the pipeline see the [CAVATICA App](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/kfdrc-somatic-variant-workflow). 
+Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation. 
+Panel data was not reprocessed and the variant files provided by the submitters were merged with our harmonized data files.
 
 ### Somatic Alteration Data
 
 #### Small Variants
 
-Multiple callers were used to determine single nucleotide variants (SNVs) since the literature suggests this reduces false positives. Using custom R scripts, a consensus SNV file was constructed, consisting only of SNVs that were called by 2 or more variant callers: GATK Mutect2, Strelka2, Lancet, and VarDict Java.  See [the consensus calling documentation](https://github.com/kids-first/kf-somatic-workflow/blob/master/docs/kfdrc-consensus-calling.md) for more detail on how the calls were combined. Annotations, including alternative gene and protein IDs and cancer references, were also added, see [the annotation calling workflow](https://github.com/kids-first/kf-somatic-workflow/blob/master/docs/kfdrc_annotation_subworkflow.md) for more details.
+Multiple callers were used to determine single nucleotide variants (SNVs) since the literature suggests this reduces false positives. Using custom R scripts, a consensus SNV file was constructed, consisting only of SNVs that were called by 2 or more variant callers: GATK Mutect2, Strelka2, Lancet, and VarDict Java. 
```

Comment:
```suggestion
Multiple callers were used to determine single nucleotide variants (SNVs) since the literature suggests this reduces false positives. 
A consensus SNV file consisting only of SNVs that were called by 2 or more variant callers: GATK Mutect2, Strelka2, Lancet, and VarDict Java was created.
[Hotspot mutations](https://www.cancerhotspots.org/#/home) were annotated and recovered if called by only 1/4 algorithms above.
```
This was done in OpenPBTA within R, but now in cavatica so removed that part.